bundles/model/org.openhab.model.script/src/org/openhab/model/script/actions/Timer.java

```diff
@@ -18,6 +18,12 @@
  * @author Kai Kreuzer
  * @since 1.0.0
  *
+ * Extended to support cronExpression for rescheduling
+ * 
+ * @author Peter Broucke
+ * @since 1.0.1
```

Comment:
please add yourself as author directly beneath Kai and leave @since unchanged (it shall indicate when the class has been introduced.